### DISCUSSION TITLE
Can I map Maven commands to ideavim? Or is there any other better solution?

### DISCUSSION THREAD

**Discussion by yaoqiangpersonal** [JetBrains/ideavim#419] [Q&A] [answered]
e.g: <Leader>mc :action mvn clean. When I use the maven command, I don't like using the mouse.

**Comment by AlexPl292** [comment, marked as answer]
Hi, you can try to use `mc :!mvn clean<CR>`.

**Reply by yaoqiangpersonal** [reply]
thanks for your reply. I try to use it. But the location isn't the directory of my workspace. Can I let it work in my workspace?

**Reply by AlexPl292** [reply]
Yes, please wait for the new IdeaVim release. It's going to be released pretty soon with this bug fixed.
